### ISSUE TITLE
ModuleNotFoundError: No module named 'Quartz' on Mac

### ISSUE THREAD

**Issue by jakubczaplicki** [boppreh/keyboard#136]
The dependency (`pyobjc`) is not being installed as part of `pip install keyboard`:

```
$ python3 -m venv venv_keyboard

$ source venv_keyboard/bin/activate

(venv_keyboard) $ pip install keyboard
Collecting keyboard
  Using cached keyboard-0.11.0-py2.py3-none-any.whl
Installing collected packages: keyboard
Successfully installed keyboard-0.11.0

(venv_keyboard) $ python
Python 3.6.4 (default, Mar  1 2018, 18:36:50) 
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.

>>> import keyboard
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jakubczaplicki/PycharmProjects/venv_keyboard/lib/python3.6/site-packages/keyboard/__init__.py", line 103, in <module>
    from. import _darwinkeyboard as _os_keyboard
  File "/Users/jakubczaplicki/PycharmProjects/venv_keyboard/lib/python3.6/site-packages/keyboard/_darwinkeyboard.py", line 3, in <module>
    import Quartz
ModuleNotFoundError: No module named 'Quartz'
>>> 
```

**Comment by glitchassassin** [comment]
There's a fix for this [in the master branch of the repository](https://github.com/boppreh/keyboard/blob/6f9961f2589cf37749239acf938a7e40ae9e9a21/setup.py#L34). It might not have been rolled out to `PyPI` yet.

**Comment by boppreh** [comment]
Fix has been rolled out now in v0.13.0. Closing this issue as duplicated, but please feel free to reopen if you still encounter the problem.

Sorry for taking so long and causing all this trouble.

**Comment by jakubczaplicki** [comment]
I am afraid the issue still exist in v.0.13.1:  

```
$ python3 -m venv venv_keyboard

$ source venv_keyboard/bin/activate

$ pip install keyboard
Collecting keyboard
  Downloading keyboard-0.13.1-py2.py3-none-any.whl (54kB)
    100% |████████████████████████████████| 61kB 722kB/s 
Installing collected packages: keyboard
Successfully installed keyboard-0.13.1
You are using pip version 9.0.1, however version 9.0.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

(venv_keyboard) $ python
Python 3.6.4 (default, Mar  9 2018, 23:15:03) 
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import keyboard
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jakubczaplicki/PycharmProjects/venv_keyboard/lib/python3.6/site-packages/keyboard/__init__.py", line 124, in <module>
    from. import _darwinkeyboard as _os_keyboard
  File "/Users/jakubczaplicki/PycharmProjects/venv_keyboard/lib/python3.6/site-packages/keyboard/_darwinkeyboard.py", line 3, in <module>
    import Quartz
ModuleNotFoundError: No module named 'Quartz'
```

**Comment by boppreh** [comment]
Note I had to undo https://github.com/boppreh/keyboard/blob/6f9961f2589cf37749239acf938a7e40ae9e9a21/setup.py#L34 because it broke installation on Windows and Linux (#143).

**Comment by boppreh** [comment]
The most likely cause of this issue is that the list of dependencies is being compiled during release, not during installation.

The environment marker format used in the fix is described in [PEP 508](https://www.python.org/dev/peps/pep-0508/#environment-markers), but I don't see what could be causing it to be misinterpreted in Windows and Linux machines (note the single quote is supposedly allowed, but something to try changing).

**Comment by boppreh** [comment]
The `map_name` problem is now fixed on version 0.13.2. Please try `pip install --upgrade keyboard` to use the latest version.
